cli/data.py

```diff
@@ -44,6 +46,17 @@ def _save_field_summary(timeseries_dataset: TimeseriesDataset, output_path: path
     _logger.info(f"Saved dataset summary to {output_path}")
 
 
+@main.command()
+@click.option("--filename", default="external_forecasts.csv")
+def update_forecasts(filename):
+    """Updates external forecasts to the current checked out covid data public commit"""
```

Comment:
Can you add a log line either at the beginning or end to signify that we're updating forecasts

Reply:
Done.